### PR TITLE
New version: TrustedRu.CryptoARMGOST version 2.5.12

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/39165)

### CODE REVIEW

File: manifests/t/TrustedRu/CryptoARMGOST/2.5.12/TrustedRu.CryptoARMGOST.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: TrustedRu.CryptoARMGOST
+PackageVersion: 2.5.12
```
Comment (ImJoakim) [inline manifests/t/TrustedRu/CryptoARMGOST/2.5.12/TrustedRu.CryptoARMGOST.installer.yaml:5]:
```suggestion
PackageVersion: 2.5
```

File: manifests/t/TrustedRu/CryptoARMGOST/2.5.12/TrustedRu.CryptoARMGOST.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: TrustedRu.CryptoARMGOST
+PackageVersion: 2.5.12
```
Comment (ImJoakim) [inline manifests/t/TrustedRu/CryptoARMGOST/2.5.12/TrustedRu.CryptoARMGOST.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.5
```

File: manifests/t/TrustedRu/CryptoARMGOST/2.5.12/TrustedRu.CryptoARMGOST.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: TrustedRu.CryptoARMGOST
+PackageVersion: 2.5.12
```
Comment (ImJoakim) [inline manifests/t/TrustedRu/CryptoARMGOST/2.5.12/TrustedRu.CryptoARMGOST.yaml:5]:
```suggestion
PackageVersion: 2.5
```

**Review by ImJoakim — COMMENTED** [review]
image

**Review by ImJoakim — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/39165#pullrequestreview-841058524

**Comment by Trenly** [comment]
Package already exists in winget
image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**